### DISCUSSION TITLE
If CoPilot doesn't have any ideas on what to put, it should tell you.

### DISCUSSION THREAD

**Discussion by Xyphyn** [community/community#6677] [Copilot Conversations] [answered]
Sometimes you write something as a comment to have Copilot give you an idea. Sometimes, it doesn't give you one, which is perfectly fine, but it should at least tell you. Several times I have thought that copilot was disabled or something, I think this would be nice.

Example:
What happens:
```
def burp():
    '''A command that does _blah blah blah_'''
```
What should happen:
```
def burp():
    '''A command that does _blah blah blah_'''
    **No autocomplete was generated. Maybe try rephrasing what you want it to do?**
```

**Comment by hsenag** [comment, marked as answer]
Thanks for the feedback! One way we do provide some feedback on what Copilot is by turning the little Copilot icon in the status bar (bottom-right) <img height="22" alt="Screenshot 2021-10-24 at 10 10 35" src="https://user-images.githubusercontent.com/451141/138587915-a92d29db-5316-4e48-99e8-7ec0cd04082e.png"> into a spinner <img height="22" alt="Screenshot 2021-10-24 at 10 11 28" src="https://user-images.githubusercontent.com/451141/138587918-7642081c-cf06-4189-a7cf-ad643d18ea7c.png"> while Copilot is thinking. If Copilot is disabled completely, the icon would be red, or not there.
